### ISSUE TITLE
docusaurus: < and > in README.md being replaced with ‹ and › in index.md

### ISSUE THREAD

**Issue by felipecrs** [tgreyuk/typedoc-plugin-markdown#148]
This is breaking comments in the README.md, causing them to not be interpreted as comments.

`README.md`:
```Markdown
<!-- toc -->
* [Usage](#usage)
* [Commands](#commands)
<!-- tocstop -->
```

`docs/api/index.md`:
```Markdown
‹!-- toc --›
* [Usage](#usage)
* [Commands](#commands)
‹!-- tocstop --›
---

**Comment by tgreyuk** [comment]
Fixed in (docusaurus-plugin typedoc 0.1.0). Now only replacing escaped angle brackets.

**Comment by felipecrs** [comment]
Thank you very much, sir!
